openapi/v2/core-metadata.yaml

```diff
@@ -552,6 +562,147 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/DeviceService'
+    ProvisionWatcher:
+      description: "A ProvisionWatcher defines the filtering criteria for device auto discovery."
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        id:
+          type: string
+          format: uuid
+          description: Uniquely identifies this request. For implementation, recommend this value be generated by the type's constructor
+        name:
+          type: string
+          description: Non-database identifier (must be unique)
+        created:
+          type: integer
+          description: Created is a timestamp indicating when the entity was created.
+        modified:
+          type: integer
+          description: Modified is a timestamp indicating when the entity was last modified.
+        labels:
+          type: array
+          description: Other labels applied to the device to help with searching
```

Comment:
Are this device labels? Rather than labels applied to the Prevision Watcher? If so the filed should be call `deviceLabels`. Here and below

Reply:
nah this should be the labels for the watcher. I'll update the description text.